docs/source/conf.py

```
@@ -176,9 +176,22 @@ def _generate_doxygen_rst(xmldir, outdir):
     _generate_groups(outdir, groups, breathe_default_project)
 
 
+def _resolve_doxygen_xml(xmldir, srcdir):
+    real_xml = os.path.abspath(os.path.join(os.path.dirname(__file__), xmldir))
+    checkout_xml = os.path.abspath(os.path.join(srcdir, xmldir))
+    if checkout_xml != real_xml:
+        checkout_root = os.path.abspath(os.path.join(srcdir, "..", ".."))
+        subprocess.run(["doxygen", "Doxyfile.in"], cwd=checkout_root, check=True)
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)